action.yml

```
@@ -50,31 +50,29 @@ runs:
         echo "GIT_AUTHOR_NAME=getsentry-bot" >> $GITHUB_ENV;
         echo "EMAIL=bot@getsentry.com" >> $GITHUB_ENV;
     - name: Craft Prepare
+      id: craft
+      shell: bash
       run: npx @sentry/craft@${{ inputs.craft_version }} prepare --no-input "${{ env.RELEASE_VERSION }}"
       env:
         # TODO: Remove this when the latest version of craft is released
         GITHUB_API_TOKEN: ${{ github.token }}
     - name: Get Release Git Info
       id: release-git-info
+      shell: bash
       run: |
         echo "::set-output name=branch::$(git rev-parse --symbolic-full-name @{-1})"
         echo "::set-output name=sha::$(git rev-parse @{-1})"
     - name: Request publish
-      if: success()
-      uses: actions/github-script@v3
-      with:
-        github-token: ${{ secrets.GH_RELEASE_PAT }}
-        script: |
-          const repoInfo = context.repo;
-          await github.issues.create({
-            owner: repoInfo.owner,
-            repo: 'publish',
-            title: `publish: ${repoInfo.repo}@${process.env.RELEASE_VERSION}`,
-            body: [
-              `Requested by: @${context.actor}`
-              ``,
-              `Quick links:`,
-              `- [View changes](https://github.com/${repoInfo.owner}/${repoInfo.repo}/compare/${{ steps.release-git-info.outputs.branch }})`,
-              `- [View check runs](https://github.com/${repoInfo.owner}/${repoInfo.repo}/commit/${{ steps.release-git-info.outputs.sha }}/checks/)`,
-            ].join('\n'),
-          });
+      shell: bash
+      run: |
+        if [[ '${{ steps.craft.outcome }}' != 'success' ]]; then
+          exit 1;
+        fi
+        title= "publish: $GITHUB_REPOSITORY@$RELEASE_VERSION"
+        body="Requested by: @$GITHUB_ACTOR
+
+        Quick links:
+        - [View changes](https://github.com/GITHUB_REPOSITORY/compare/${{ steps.release-git-info.outputs.branch }})
```

Assuming `GITHUB_REPOSITORY` is an env var? Same below.

```suggestion
        - [View changes](https://github.com/$GITHUB_REPOSITORY/compare/${{ steps.release-git-info.outputs.branch }})
```